api/src/main/java/org/openmrs/module/pihmalawi/PihMalawiConstants.java

```diff
@@ -5,6 +5,7 @@ public class PihMalawiConstants {
 
     public static final String MODULE_ID = "pihmalawi";
     public static final String OPENMRS_WAREHOUSE_CONNECTION_PROPERTIES_FILE_NAME = "warehouse-connection.properties";
+    public static final String REST_DIGITAL_SIGNATURE_PUBLIC_KEY = "PUBLIC_KEY";
```

Comment:
If this is a runtime property name, let's make it more consistent with approaches used there.  Something like `pihmalawi.syncPublicKeyFile` would seem more appropriate.